webui/src/Main.elm

```diff
@@ -179,22 +173,6 @@ update msg model =
                 ( { model | surveyResponseModel = surveyResponseModel }, Cmd.map SurveyResponseMsg cmd )
 
 
-getHTTPErrorMessage : Http.Error -> String
-getHTTPErrorMessage error =
```

Comment:
If this one is really not being called, I would like to make sure we start using something similar or better in order to show errors from failed HTTP requests. I had hooked up some snackbar code to show when we get 401 errors, it would be interesting to see if that error display code could be improved by using this function or something even better.

Reply:
It is not called in Main, so I removed it.

I did a quick string search and it looks like Main doesn't handle any errors.

Currently the shared getHTTPErrorMessage effectively calls `console.error()` and opens the snackbar.

A better thing would be to show an overlay depending on the error type.

`type ErrorType = Http | GenericResult | Other`

or something of that mindset. then we could have a type-safe way to show a banner/snackbar/popup/overlay.